_main.scss

```diff
@@ -0,0 +1,68 @@
+.section-one-main {
+    padding: 2rem 1.8rem;
+    border-left: 2px solid $color-primary;
+    border-right: 2px solid $color-primary;
+    &__heading {
+        padding: 0;
+        border-bottom: 2px solid $color-primary;
+    }
+    &__input {
+        height: 3rem;
+        border-radius: 2rem;
+    }
+    &__list-button {
+        display: grid;
+        grid-template-columns: auto 1fr;
+        align-items: center;
+    }
+    ul {
+        list-style: none;
+        display: grid;
+        grid-template-columns: 2rem 2rem 2rem 2rem 2rem 10rem;
```

Comment:
could you potentially use a repeat() function here? eg. `repeat(5, 2rem) 10rem;`

Reply:
Much more elegant. Thanks.